### PR TITLE
refs #13 

### PR DESCRIPTION
**Pull request by emullaney**
new custom template tag make_unique for filtering out duplicates in m2ms

### CODE REVIEW

File: portal_pages/templatetags/portal_extras.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+@register.filter
+def make_unique(value, unique_var): # Only one argument.
+    """Converts a string into all lowercase"""
```
Comment (kmtracey) [inline portal_pages/templatetags/portal_extras.py]: No, that's not what this does!

**Comment by kmtracey** [comment]
Looks good, but we should include some test(s) for the new template filter before merging.

**Comment by emullaney** [comment]
Will work on tests, also need to check into "Preview" mode.

**Comment by kmtracey** [comment]
We generally go with a tests directory rather than a single tests.py file (see the accounts app for a structure). Also if you use the UserFactory defined in the accounts tests directory then the setup code here can be simplified to focus on just the duplicated name field rather than having to fully-specify the other fields for the user which really don't matter....

**Comment by emullaney** [comment]
@kmtracey this PR is ready for your perusal... however, for some reason now that I'm using the "tests" folder inside of the "portal_pages" app, now the usual "python manage.py test" command doesn't find this test. Can you tell why?

I was able to test this code by typing out the full "python manage.py test portal_pages.tests so that this test would be found.

File: portal_pages/tests/factories.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        the user's username (email address).
+        """
+        manager = cls._get_manager(target_class)
+        username = kwargs.pop('username')
+        password = kwargs.pop('password')
+        user = manager.create_user(username, password)
+        for key, value in kwargs.items():
+            setattr(user, key, value)
+        user.save(update_fields=kwargs.keys())
+        return user
```
Comment (kmtracey) [inline portal_pages/tests/factories.py]: We should not duplicate this factory code in 2 apps, portal_pages tests should just use the accounts tests factory...

**Comment by kmtracey** [comment]
I believe the autodiscovery isn't working due to no **init**.py file in the portal_pages tests directory, try adding that and see if it works better...

**Comment by kmtracey** [comment]
:+1:

**Pull request merged**